app/src/org/commcare/views/widgets/QuestionWidget.java

```diff
@@ -624,7 +625,10 @@ public boolean hasListener() {
     }
 
     public void checkFileSize(File file){
-        if (FileUtil.isFileOversized(file)) {
+        if (FileUtil.isFileToBigToUpload(file)) {
+            long overByAmount = (file.length() - FormUploadUtil.MAX_BYTES) / 1024;
```

Comment:
I figured it would be useful to see some sort of feedback about how much over the attachment is. Open to taking it out or showing a more comprehensive message like in https://github.com/dimagi/commcare-android/pull/1446